backend/src/db/migrations/20230826083159_add_user_roles_table.ts

```diff
@@ -0,0 +1,53 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'user_roles';
+
+const COLUMN_NAME = {
+  ID: 'id',
+  NAME: 'name',
+  KEY: 'key',
+  CREATED_AT: 'created_at',
+  UPDATED_AT: 'updated_at',
+} as const;
+
+const USER_NAMES = {
+  CHATBOT: 'Chatbot',
+  USER: 'User',
+} as const;
+
+const USER_ROLES = {
+  CHATBOT: 'chatbot',
+  USER: 'user',
+} as const;
+
```

Comment:
```suggestion
const ROLE_NAME = {
  CHATBOT: 'Chatbot',
  USER: 'User',
} as const;

const ROLE_KEY = {
  CHATBOT: 'chatbot',
  USER: 'user',
} as const;

```

Reply:
Due to the comment from Liza, I deleted them!